src/widgets/list.rs

```diff
@@ -178,6 +178,13 @@ impl ListState {
 /// let item = ListItem::new("Item 1");
 /// ```
 ///
+/// Anything that can be converted to [`Text`] can be a [`ListItem`].
```

Comment:
Just as a stylistic thing, I prefer to leave a newline between text and code fences (there's a default markdown lint error that picks this up for actual markdown files). It's not a big deal, mainly a consistency thing.

Reply:
I don't mind adding a newline, I prefer consistency.
What markdown linter/formatter do you use? I hate formatting manually and checking every single line I write.